examples/direct-upload-to-gcs/function/go.mod

```diff
@@ -0,0 +1,9 @@
+module imageuploader
```

Comment:
module should be the full path, starting with `github.com/`

Reply:
done